upgrade-guides/v2.0.0.md

```diff
@@ -207,13 +207,26 @@ const DeepComponent = React.createClass({
 }
 
 // v2.0.0
-// Use context.router
+// You have a couple options:
+// 1) Use context.router (especially if on the server)
 const DeepComponent = React.createClass({
   contextTypes: {
     router: React.PropTypes.object.isRequired
   },
-  someHandler() {
-    this.context.router.push(...)
+  componentWillMount() {
```

Comment:
Can we make this an `onClick` handler?

"Mount" hooks are actually not a great example, since you should usually use `onEnter` to handle this sort of thing.